docs/sources/visualizations/gauge-panel.md

```diff
@@ -53,3 +53,10 @@ Adjust the sizes of the gauge text.
 
 - **Title -** Enter a numeric value for the gauge title size.
 - **Value -** Enter a numeric value for the gauge value size.
+
+## Standard options
+
+Adjust various options.
+
+- **Min/Max -** If the values of min and max are additive inverse numbers (e.g. min=-500, max=500), then the neutral point of the gauge is put at 12 o'clock. Negative values are plotted to the left side, positive values to the right side.
```

Comment:
Aiming for clarity with plainer language here, while understanding the limitations in the comments:

```suggestion
- **Min/Max -** Define a range to gauge, with the neutral point plotted at the center of the gauge. Values higher or lower than the neutral point plotted to the left or right of center, respectfully. The values of min and max must be opposite numbers; for instance, if the min is -500, the max must be 500.
```